### DISCUSSION TITLE
Welcome to stripe-sdk Discussions!

### DISCUSSION THREAD

**Discussion by ezet** [ezet/stripe-sdk#120] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bhaveshcr** [comment]
Hi, thanks for sharing this code. I am referring to  "stripe-sdk/lib/src/stripe.dart" file. How to initialize Stripe to have direct payments on connected accounts. If StripeAccount is the customer account, how to setup direct payment StripeAccount header as suggested here:
 https://stripe.com/docs/connect/authentication#adding-the-connected-account-id-to-a-client-side-application
